modules/oc-mirror-workflows-delete-v2.adoc

```diff
@@ -54,6 +54,49 @@ oc-mirror plugin v2 deletes only the manifests of the images, which does not red
 
 To free up storage space from unnecessary images, such as those with deleted manifests, you must enable the garbage collector on your container registry. With the garbage collector enabled, the registry will delete the image blobs that no longer have references to any manifests, reducing the storage previously occupied by the deleted blobs. The process for enabling the garbage collector differs depending on your container registry.
 
+The process to free up storage running the garbage collector may not work as expected due to a known issue in the distribution registry. This issue does not occur when using {quay}.
```

Comment:
I'm not sure the _by_ is the right word to add; but something is needed here for clarity, I think. 
```suggestion
The process to free up storage by running the garbage collector might not work as expected due to a known issue in the distribution registry. This issue does not occur when using {quay}.
```